.travis/build.sh

```diff
@@ -82,20 +82,24 @@ if [ "${MAIN_BUILD}" == "TRUE" ] ; then
     EXIT=$?
     exitIfError
 
-    clearDockerEnv
-    mvn -e -V -B clean install -f testsuite -Pkafka-3_1_2
-    EXIT=$?
-    exitIfError
-
-    clearDockerEnv
-    mvn -e -V -B clean install -f testsuite -Pkafka-3_0_0
-    EXIT=$?
-    exitIfError
-
-    clearDockerEnv
-    mvn -e -V -B clean install -f testsuite -Pkafka-2_8_1
-    EXIT=$?
-    exitIfError
+    if [ $JAVA_VERSION -le 11 ]; then
```

Comment:
Why would you run the tests only on Java 11?

Reply:
Those profiles use Kafka images with Java 11 so they are unable to use the OAuth jar built with Java 17 and latest class version of Java 17. That's why they have to be skipped for Java 17 build.

Reply:
Makes sense, thanks. GitHub didn't show the tests above these lines running on all versions so I did not noticed them.